R/codemeta.R

```diff
@@ -49,23 +49,26 @@ create_cm <- memoise::memoise(.create_cm)
 #' Create the codemetas for all files
 #'
 #' @param old_cm path to latest CodeMeta version
+#' @param folder folder under which the
 #'
 #' @return A JSON codemeta
 #' @export
-create_codemetas <- function(old_cm = NULL){
+create_codemetas <- function(old_cm = NULL, folder = "repos"){
   if(!is.null(old_cm)){
     old_cm <- jsonlite::read_json(old_cm)
     old_cm <- old_cm[lengths(old_cm) > 0]
   }
+  list_repos <- function(directory) {
```

Comment:
No longer hard-coding that "repos" contains "ropenscilabs", "ropensci" and "others", so it might be easier to add "ropensci-archive".